typedb/TypeDBSimulation.kt

```diff
@@ -73,9 +80,58 @@ abstract class TypeDBSimulation<out CONTEXT: Context<*, *>> protected constructo
         }
     }
 
+    private fun splitSchemaQueries(queryString: String): List<String> {
```

Comment:
I'm pretty confident we could replace this function with a call to `TypeQL.parseQueries`. It does all this, though we'll need to do a filter to remove all the queries that aren't `define` or `undefine`.

Reply:
Done.